README.md

```diff
@@ -193,3 +193,27 @@ If you register the service providers in the correct order in your container (A
 - [Simplex](https://github.com/mnapoli/simplex): A [Pimple 3](https://github.com/silexphp/Pimple) fork with full [container-interop](https://github.com/container-interop/container-interop) compliance and cross-framework service-provider support.
 - [Service provider bridge bundle](https://github.com/thecodingmachine/service-provider-bridge-bundle): Use container-interop's service-providers into a Symfony container.
 
+##FAQ
+
+####Why static methods?
```

Comment:
FYI it's not a problem with Github but some markdown renderers have trouble with titles with no space after the `#`. I'm not sure if it's valid CommonMark. Anyway it's not an issue.